R/relevel.R

```diff
@@ -13,12 +13,25 @@
 #' @template step-return
 #' @family dummy variable and encoding steps
 #' @export
-#' @details The selected variables are releveled to a level
+#' @details
+#'
+#' The selected variables are releveled to a level
 #' (given by `ref_level`). Placing the `ref_level` in the first
 #' position.
```

Comment:
This isn't related to this PR, but I don't know what this sentence means.

Reply:
`step_relevel()` changes what the first level of a factor is:

``` r
library(recipes)

iris$Species |>
  head()
#> [1] setosa setosa setosa setosa setosa setosa
#> Levels: setosa versicolor virginica

recipe(~., data = iris) |>
  step_relevel(Species, ref_level = "virginica") |>
  prep() |>
  bake(new_data = NULL) |>
  pull(Species) |>
  head()
#> [1] setosa setosa setosa setosa setosa setosa
#> Levels: virginica setosa versicolor
```